app/models/header_lookup.rb

```diff
@@ -11,8 +11,8 @@ class HeaderLookup
   MEMOIZED_MESH_FILE = "memoized_mesh.txt"
   MEMOIZED_LCSH_FILE = "memoized_lcsh.txt"
   MEMOIZED_LCNAF_FILE = "memoized_lcnaf.txt"
-  SEARCHABLE_MESH_FILE = "subjects_mesh.yml"
-  SEARCHABLE_LCSH_FILE = "subjects_lcsh.yml"
+  SEARCHABLE_MESH_FILE = ENV['SUBJECTS_MESH_FILE']
+  SEARCHABLE_LCSH_FILE = ENV['SUBJECTS_LCSH_FILE']
```

Comment:
minor: Is there a reason those aren't hardcoded like "memoized_mesh.txt"? 

In InvenioRDM deployments, these kinds of environment variables can come from multiple places typically split between common (not site or secret specific configurations) and site/secret specific configurations. `ENV['SUBJECTS_MESH_FILE']` doesn't seem site/secret specific.